format/src/main/scala/org/scalafmt/internal/SyntaxTokens.scala

```diff
@@ -282,6 +268,112 @@ object SyntaxTokens {
     def tokensWith: KwWith = tree.findBetween[KwWith](_.lhs, _.rhs).get
   }
 
+  // == Defn ==
+  implicit class XtensionDefnClassSyntax(val tree: Defn.Class) extends AnyVal {
+    def tokensClass: KwClass = tree.find[KwClass].get
+
+    // tparams
+    def tokensLeftBracket: Option[LeftBracket] =
```

Comment:
I think it would be more handy to have `tokensBrackets: Option[(LeftBracket, RightBracket)]` and the implementation could use `MatchingParentheses` to simplify things. The current implementation make no guarantee that the left bracket matches the right bracket.

Reply:
`tokensTypeParamsBrackets` might be clearer